src/Tribe/Importer/Admin_Page.php

```diff
@@ -159,19 +160,24 @@ public function render_csv_tab() {
 						$header[] = $letter++;
 					}
 				}
+
 				$import_type = get_option( 'tribe_events_import_type' );
 				$messages = $this->errors;
 				include Tribe__Events__Importer__Plugin::path( 'src/io/csv/admin-views/columns.php' );
+
```

Comment:
These are just misc formatting tweaks unrelated to the main thrust of the PR right? How come the extra line before breaking here, but not in the case of the subsequent case statements?